packages/server/lib/updater.js

```diff
@@ -1,30 +1,37 @@
+const os = require('os')
 const debug = require('debug')('cypress:server:updater')
 const semver = require('semver')
 const rp = require('@cypress/request-promise')
 const pkg = require('@packages/root')
 const { agent } = require('@packages/network')
 const konfig = require('./konfig')
 const { machineId } = require('./util/machine_id')
-
-const _getManifest = (id) => {
+const _getManifest = ({ id, initialLaunch, testingType }) => {
   const url = konfig('desktop_manifest_url')
 
   return rp.get({
     url,
     headers: {
       'x-cypress-version': pkg.version,
+      'x-os-name': os.platform(),
       'x-machine-id': id,
+      'x-initial-launch:': initialLaunch,
```

Comment:
headers are strings, i'm sure this is done somewhere down the line, but just to make sure the download server gets what it expects (either "true" or not "true"):

```suggestion
      'x-initial-launch:': String(initialLaunch),
```

Reply:
wait until you see the request-attempt header PR...